app/inputs/price_input.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# Adds input followed by currency string and displays only two
+# decimals of the value.
+#
+# FIXME: if the price has more than 2 decimals than they are cut
+# on display - clicking save then damages the data!
```

Comment:
Is this different than before?

Reply:
Nothing has changed as far as I know but I haven't looked into this TBH, it's just copied like it was.